system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -959,6 +959,46 @@ component {
 		}
 	}
 
+	public string function cloneForm(
+		  required string basedOnFormId
+		, required string name
+		, required string description
+	) {
+		var originalFormData = getForm( id=arguments.basedOnFormId );
+		var cloneFormData = structNew();
```

Comment:
Simplified the code while looping 'originalFormData.columnList' by 
<code>var cloneFormData = { name=arguments.name, description=arguments.description };</code>